hack/do-janitor/do-janitor.go

```diff
@@ -104,6 +104,21 @@ func main() {
 		}
 	}
 
+	keys, err := keyList(ctx, client)
+	if err != nil {
+		log.Fatalf("failed to list keys: %+v", err.Error())
+	}
+
+	for _, key := range keys {
```

Comment:
we need to filter the keys

we have some keys there that should stay there.

we can delete keys that start with `capdo`